client/galaxy/scripts/components/RuleCollectionBuilder.vue

```diff
@@ -446,7 +446,7 @@
                         :data="hotData.data"
                         :col-headers="colHeadersDisplay"
                         :read-only="true"
-                        stretch-h="all"
+                        :stretch-h="all"
```

Comment:
I'm not sure I understand this one; was this not binding correctly before?

Reply:
I noticed it as an inconsistency and thought it could be part of the problem, so I decided to fix it while I was there.

Reply:
Consistency is important, but this isn't the same thing as binding, for example `:col-headers="colHeadersdisplay"`.  The colon is shorthand for setting up v-bind, which is used to dynamically bind attributes to an expression.  This is just a static attribute we want to set.